.travis.yml

```diff
@@ -1,24 +1,25 @@
 language: ruby
-sudo: false
+os: linux
```

Comment:
why this change?

Reply:
i was validating the contents of this file on https://config.travis-ci.com/explore, where I found suggestions to make these changes

Reply:
ok, I see 

```
[warn] on root: deprecated key: "ruby_supported_versions" (anchor on a non-private key)

[warn] on root: deprecated key: "jruby_supported_versions" (anchor on a non-private key)

[warn] on root: deprecated key: "jdk_supported_versions" (anchor on a non-private key)

[warn] on root: deprecated key: "rails_supported_versions" (anchor on a non-private key)

[warn] on root: deprecated key: "sudo" (The key `sudo` has no effect anymore.)

[info] on root: the key matrix is an alias for jobs, using jobs

[info] on env: the key matrix is an alias for jobs, using jobs

[info] on root: missing os, using the default "linux"

```

Is out of scope, I guess

Reply:
you mean that we move this change to another PR?

Reply:
yup.